src/OrbitQt/main.cpp

```diff
@@ -66,6 +67,8 @@ ABSL_DECLARE_FLAG(bool, local);
 ABSL_DECLARE_FLAG(bool, devmode);
 ABSL_DECLARE_FLAG(bool, nodeploy);
 
+ABSL_FLAG(bool, clear_source_paths_mappings, false, "Clear all the stored sources paths mappings");
```

Comment:
I wonder if we should have a comment that states that the main (only?) intention for usage of this flag is as part of e2e tests?

Reply:
A user-facing comment, or for us in the code?

Reply:
For us in the code.

Reply:
I added a comment.

Reply:
I will go ahead and merge this.